Source/SwiftLintBuiltInRules/Rules/Lint/BlanketDisableCommandRule.swift

```diff
@@ -4,7 +4,9 @@ struct BlanketDisableCommandRule: Rule, SourceKitFreeRule {
     static let description = RuleDescription(
         identifier: "blanket_disable_command",
         name: "Blanket Disable Command",
-        description: "swiftlint:disable commands should be re-enabled before the end of the file",
+        description: "'swiftlint:disable' commands should use 'next', 'this' or 'previous' to disable rules for a " +
+                     "single line, or 'swiftlint:enable' to re-enable the rules immediately after the violations " +
+                     "to be ignored",
```

Comment:
That's what's dumped into the documentation. Here, you can elaborate. You may use multiple sentences and explain the rule in more detail.

Since that's going to be interpreted as Markdown, you may put code into backticks.